dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -127,6 +128,7 @@ class ScriptSeedTest < ActiveSupport::TestCase
       json = ScriptSeed.serialize_seeding_json(script)
       script.lessons.each {|l| l.resources.destroy_all}
       script.lessons.each {|l| l.vocabularies.destroy_all}
+      script.lessons.each {|l| l.programming_expressions.destroy_all}
```

Comment:
remove this line -- only vocab and resources are prevented from being seeded in this scenario where the course version is not yet present.